sdk/js/src/Client.ts

```diff
@@ -51,18 +51,19 @@ export class DVCClient implements Client {
         if (options?.reactNative) {
             this.store = new ReactNativeStore(this.logger)
         } else {
-            this.store = new Store(typeof window !== 'undefined' ? window.localStorage : stubbedLocalStorage, this.logger)
+            this.store = new Store(
+                typeof window !== 'undefined' ? window.localStorage : stubbedLocalStorage, this.logger
+            )
         }
 
-        if (!user.isAnonymous) {
+        if (user.user_id || !!!user.isAnonymous) {
```

Comment:
`!!!undefined` === `true`, so `anonUserId` will be removed when user didn't provide both `user_id` and `isAnonymous`

Reply:
just updated this :) take a look again please! 🙏